api/scpca_portal/config/logging.py

```diff
@@ -50,3 +54,42 @@ def get_and_configure_logger(name: str) -> logging.Logger:
     logger.logger.addHandler(handler)
 
     return logger
+
+
+def get_formatted_time(timestamp: float) -> str:
+    """Format a timestamp to a local time."""
+    return datetime.fromtimestamp(timestamp).strftime("%H:%M:%S")
+
+
+def log_func_run_time(logger):
+    """
+    Log start time, end time, and duration of the wrapped function.
+    """
+
+    def decorator(func):
+        @wraps(func)
+        def wrapper(*args, **kwargs):
+            global LOG_FUNC_RUN_TIMES
+            if not LOG_FUNC_RUN_TIMES:
+                return func(*args, **kwargs)
+
+            start_time = time.time()
+            logger.info(f"\nStart of function '{func.__module__}::{func.__name__}'.")
+
+            try:
+                result = func(*args, **kwargs)
+            finally:
+                end_time = time.time()
+                duration = end_time - start_time
+                logger.info(
+                    f"\nEnd of function '{func.__module__}::{func.__name__}'.\n"
+                    f"Start time: {get_formatted_time(start_time)}, "
+                    f"End time: {get_formatted_time(end_time)}, "
```

Comment:
```suggestion
                    f"Start time: {get_formatted_time(start_time)}, "
```

Reply:
I actually think that having both the start and end times in succession makes the log more readable. I updated how the times are presented, though, so that it would look cleaner (see updated screenshot in the PR description).